[Feature] Implement Additional Evil Teams

Blitz425 opened the pull request:
## What are the changes?
Added Aether Foundation and Team Skull from Pokemon Sun/Moon, and Macro Cosmos from Pokemon Sword/Shield

## Why am I doing these changes?
Heavily requested feature from feature-vote in the discord, this additionally aims to add even more replayability and randomness factor to PokeRogue and give the player more references / love from the mainline titles to look forward to in their PokeRogue experience. 

## What did change?
- Added Aether/Skull/Macro as possible Evil Team Spawns.
- Added Admins from Aether/Skull/Macro on waves 66 and 114. Unlike other admins, since these 3 are the only admins in their respective teams and have their own signature Pokemon separate from other grunts in their game, they instead pull from a pool of Pokemon that fit their general theming or have used in the mainline titles.
- New dialogue, BGM, and sprites.
- Magma Grunt / Admin pool loses **Turtonator** for **Capsakid** to give **Turtonator** to Aether Grunts, loses Swinub for Rolycoly for better thematics
- Minor changes to other Evil Leaders for QoL or to redistribute to the new ones:
    - Archie 2 - **Kingdra** replaced with **Empoleon** as it no longer has Drizzle, **Gorebyss/Huntail** replaced with **Overqwil** for better coverage.
    - Cyrus 1 - **Overqwil** replaced with **Gliscor** to keep the bat theming of **Crobat** and give to Archie.
    - Ghetsis 2 - Given **Basculegion/Jellicent** and **Genesect** over **Darmanitan** / **Durant** slots to round out the team better, replaced **Iron Moth** with base **Volcarona** to give to Lysandre and no longer the boss being replaced by **Genesect**.
    - Lysandre 1 - Replaced Milotic with **Clawlitzer** / **Dragalge** for more variation and to give **Milotic** to Lusamine, given **Malamar** as a variation for **Heliolisk** for different coverage and to give a thematic evil pokemon from his generation. 
    - Lysandre 2 - Replaced **Gholdengo** / **Aegislash** with **Iron Moth** to give them to Rose.

**[Subject to change]**

Lusamine 1:
- Clefable
- Milotic / Primarina
- Lilligant / Hisui Lilligant
- Galar Slowbro / Galar Slowking
- Bewear / Lopunny
- **Nihilego**

Lusamine 2:
- **Clefable**
- Milotic / Primarina
- Pheromosa
- Stakataka / Celesteela / Guzzlord
- **Nihilego**
- **Necrozma**

Guzma 1:
- Lokix / Yanmega
- Heracross
- Scizor / Kleavor
- Galvantula / Vikavolt
- Pinsir-Mega
- **Golisopod**

Guzma 2:
- **Golisopod**
- **Buzzwole**
- Hisui Samurott / Crawdaunt
- Scizor / Kleavor
- Pinsir-Mega
- **Xurkitree**

Rose 1:
- Archaludon
- Ferrothorn / Escavelier
- Sirfetch'd / Mr. Rime
- Corviknight
- Perrserker / Klinklang
- **Copperajah-Gmax**

Rose 2:
- **Archaludon**
- Aegislash / Gholdengo
- Dracovish / Dracozolt
- Melmetal
- **Galar Articuno/Galar Zapdos/Galar Moltres**
- **Copperajah-Gmax**
### Screenshots/Videos

![image](https://github.com/user-attachments/assets/5120ca20-246b-47d8-8bdc-3515c5752c95)

![image](https://github.com/user-attachments/assets/d6c9a6c4-80f8-4377-b8e6-68dda25528df)

![image](https://github.com/user-attachments/assets/78d825ee-da7f-48ef-8f53-3cdd42e90ae0)

![image](https://github.com/user-attachments/assets/ef223e97-165e-491a-9c38-afc086db81bb)

![image](https://github.com/user-attachments/assets/a2541c9b-bf93-40e5-8d36-b930a6a5a644)

![image](https://github.com/user-attachments/assets/efcb8de2-9235-4640-843e-1183d74df133)


## How to test the changes?
Start a classic run and check out the new encounters on floors 35, 62, 64, 66 ,112, 114, 115, and 165
You can also use overrides to jump to those floors

## Checklist
- [x] I'm using `beta` as my base branch
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [ ] Have I considered writing automated tests for the issue?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
    - [x] Have I provided screenshots/videos of the changes?

Tempo-anon commented:
Please post a summary of the team pools and any other changes in the PR description so it's easy at a glance for reviewers to see what's new.

Adri1 commented:
Would it also be possible to update French locales woth the last updates made in beta + adding the names of the new trainers as you did for other languages?

Adri1 commented:
Don't forget to also update trainers.ts files with the recently added **Flare Admin** between Flare Grunt and Aether Foundation Employee 👌

Adri1 commented on src/locales/fr/trainers.ts:
```diff
@@ -265,7 +277,17 @@ export const trainerNames: SimpleTranslationEntries = {
+  "cyrus": "Hélio",
+  "ghetsis": "Ghetis",
+  "lysandre": "Lysandre",
+  "faba": "Faba",
+  "lusamine": "Lusamine",
+  "plumeria": "Plumeria",
+  "guzma": "Guzma",
+  "oleana": "Oleana",
+  "rose": "Rose",
```
```suggestion
  "faba": "Saubohne",
  "lusamine": "Elsa-Mina",
  "plumeria": "Apocyne",
  "guzma": "Guzma",
  "oleana": "Liv",
  "rose": "Shehroz",
```

Adri1 commented on src/locales/fr/trainers.ts:
```diff
@@ -19,6 +19,12 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Leader de la Team Galaxie",
   "plasma_boss": "Leader de la Team Plasma",
   "flare_boss": "Leader de la Team Flare",
+  "aether_boss": "Aether President",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_boss": "Team Skull Boss",
+  "skull_admin": "Team Skull Admin",
+  "macro_boss": "Macro Cosmos President",
+  "macro_admin": "Macro Cosmos",
```
```suggestion
  "aether_boss": "Présidente d’Æther",
  "aether_admin": "Fondation Æther",
  "skull_boss": "Boss de la Team Skull",
  "skull_admin": "Admin Team Skull",
  "macro_boss": "Président de Macro Cosmos",
  "macro_admin": "Macro Cosmos",
```

Tempo-anon commented:
```
Lusamine 1:
- Milotic / Primarina
- Lilligant / Hisui Lilligant
- Clefable
- Espathra
- Bewear / Lopunny
- **Nihilego**
```
I like Bewear and I like Mega Lopunny but I'm not as big of a fan of just Lopunny. I also am not sure I like Nihilego right in the first team. If you wanted to keep the theme of boss1 having a mega then replacing Nihilego with Mega Lopunny would be perfect.

```
Lusamine 2:
- **Nihilego**
- Milotic / Primarina
- Clefable
- Stakataka / Celesteela
- **Pheromosa**
- **Necrozma**
```
Was Stakataka chosen as a variant because of typing? I would have personally gone for something like Naganadel (a bulky UB and a fast offensive UB provide different team roles but fulfill similar team thematic roles). Necrozma is....fine objectively speaking because it's just 600BST but somehow it seems to feel like it has more impact because it has more powerful forms. Did you end up ditching the idea of Flutter Mane and Scream Tail here? Are Milotic/Primarina/Clefable strong enough or thematic enough to justify carrying over 2 team slots from the first fight?

```
Guzma 1:
- Golisopod
- Lokix / Yanmega
- Heracross
- Scizor / Kleavor
- Galvantula / Vikavolt
- **Pinsir-Mega**
```
I would have maybe liked to see Masquerain as an variant of Yanmega for typing and it's on his team, but to be honest Masquerain's power level is pretty low. No complaints about the rest of the team.

```
Guzma 2:
- **Golisopod**
- Buzzwole
- Hisui Samurott
- Scizor / Kleavor (Made a 1 bar boss to make up for lack of power compared to other leaders)
- **Pinsir-Mega**
- **Xurkitree**
```
Buzzwole not being a boss feels like a crime and would bump up the power level a lot. No reason for Goliospod to take its boss slot especially since it's not even a boss in the first fight. Not a fan of 1 segment bars because that just sticks out. Hisui Samurott also feels a little random on the team. In terms of power, I guarantee you that 3 bosses of Mega Pinsir, Xurkitree, and Buzzwole is on par with the other power levels.

```
Rose 1:
- Archaludon
- Ferrothorn / Escavelier
- Tinkaton
- Corviknight / Skarmory
- Perrserker / Klinklang
- **Copperajah-Gmax**

Rose 2:
- **Archaludon**
- Empoleon
- Aegislash / Gholdengo
- Iron Crown / Cobalion
- **Melmetal**
- **Copperajah-Gmax**
```
Not much to say about Rose. Seems fine to me.

```
Minor changes to other Evil Leaders for QoL or to redistribute to the new ones:
    - Archie 2 - **Gorebyss/Huntail** replaced with **Overqwil** for better coverage.
    - Cyrus 1 - **Overqwil** replaced with **Gliscor** to keep the bat themeing of **Crobat**.
    - Ghetsis 2 - Given **Basculegion** and **Genesect** over **Darmanitan** / **Durant** slots to round out the team better, replaced **Iron Moth** with base **Volcarona** to give to Lysandre and no longer the boss being replaced by **Genesect**.
    - Lysandre 2 - Replaced **Gholdengo** / **Aegislash** with **Iron Moth** to give them to Rose.
 ```
Archie 2: I still personally really like Gorebyss/Huntail for the Hoenn connection but Overqwil is fine I guess
Cyrus 1: Ok sure I guess
Ghetsis 2: Replacing Durant with Genesect... it's fine I guess though I often see Genesect more associated with Colress. If you're going to be removing the paradox moths from Ghetsis you might as well also tweak his first team because that was part of the reason I had them on the second team. I also still like Darm/GDarm because they both hit very hard and have Unovan roots. My biggest complaint is that with your proposed changes there's only a single variation on his team now and that's Hydreigon/Iron Jugulis and removing so much variety is not good in my opinion
Lysandre: I can be convinced for thematic Iron Moth (it is similar to the drones he uses) but losing variety still feels bad, but it's not as bad as Ghetsis 2

CodeTappert commented on src/locales/de/trainers.ts:
```diff
@@ -19,6 +19,12 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Galaktik-Boss",
   "plasma_boss": "Weiser von Team Plasma", // This is on purpose, since "Ghetsis" is never mentioned as the boss of team plasma in the game but as "Weiser"
   "flare_boss": "Flare-Boss",
+  "aether_boss": "Æther-Präsidentin",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_boss": "Skull-Boss",
+  "skull_admin": "Team Skull Admin",
+  "macro_boss": "Geschäftsführer von Macro Cosmos",
+  "macro_admin": "Macro Cosmos"
```
```suggestion
  "aether_boss": "Æther-Präsidentin",
  "aether_admin": "Æther-Regionalleiter",
  "skull_boss": "Skull-Boss",
  "skull_admin": "Team Skull Vorstand",
  "macro_boss": "Geschäftsführer von Macro Cosmos",
  "macro_admin": "Vizepräsidentin von Macro Cosmos"
```

Note: The Title for Faba & Oleana are for their respective characters. They probably would not fit if we ever add other admins.

CodeTappert commented on src/locales/de/trainers.ts:
```diff
@@ -276,6 +288,12 @@ export const trainerNames: SimpleTranslationEntries = {
   "cyrus": "Zyrus",
   "ghetsis": "G-Cis",
   "lysandre": "Flordelis",
+  "faba": "Faba",
+  "lusamine": "Samantha",
+  "plumeria": "Plumeria",
+  "guzma": "Bromley",
+  "oleana": "Oleana",
+  "rose": "Rose",
```
```suggestion
  "faba": "Fabian",
  "lusamine": "Samantha",
  "plumeria": "Fran",
  "guzma": "Bromley",
  "oleana": "Olivia",
  "rose": "Rose",
```

40chyan commented on src/locales/zh_CN/dialogue.ts:
```diff
@@ -562,6 +642,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
+    "encounter": {
+      1: `Doesn't matter who I'm up against, I'm gonna beat them down!
+          $That's what big bad Guzma is all about!`
+    },
+    "victory": {
+      1: "Guzma!!! What is wrong with you?!"
+    },
+    "defeat": {
+      1: "Y'all are stupid!"
+    }
+  },
+  "macro_boss_rose_1": {
+    "encounter": {
+      1: `I must provide limitless energy to ensure everlasting prosperity for everyone...
+          $It is my purpose, my duty, my destiny!`
+    },
+    "victory": {
+      1: `You still don't understand, trainer...
+          $We... No, I am going to change the course of history!`
+    },
+    "defeat": {
+      1: "You still don't understand a thing!"
+    }
+  },
+  "macro_boss_rose_2": {
+    "encounter": {
+      1: `I'm committed to solving the energy problem in the Galar region—and, of course, around the world.
+          $My experience and accomplishments that made Macro Cosmos a success are proof my methods work.
+          $I don't intend to change my mind, even if I lose.`
+
+    },
+    "victory": {
+      1: `I'd forgotten how great Pokémon battles are! It's been so long since I battled...
+          $That sure was satisfying, I accept defeat for this battle.`
+    },
+    "defeat": {
+      1: `I suppose it must seem that I am doing something terrible. I don't expect you to understand.
+          $But I must provide the Galar region with limitless energy to ensure everlasting prosperity.`
+    }
+  },
```
```suggestion
      1: "你吓到我可爱的异兽了！我要让你先闭嘴！"
    },
    "victory": {
      1: "你…你怎么这么不听话！"
    },
    "defeat": {
      1: "哼嗯…"
    }
  },
  "aether_boss_lusamine_2": {
    "encounter": {
      1: `为什么你要一直妨碍我！\n我受够你了，完完全全受够了！ 
          $不用说废话了…有了虚吾伊德的力量。\n我会让你明白，你来这里是有多么错误！`
    },
    "victory": {
      1: "呃呃啊啊啊啊呜呜呜啊啊！"
    },
    "defeat": {
      1: "我只要和我可爱的异兽在一起就好了！你们怎么样都无所谓了！"
    }
  },
  "skull_boss_guzma_1": {
    "encounter": {
         1: `摧毁，摧毁，\n不管摧毁几次也从不手软
          $惹人厌的古兹马大爷在此。`
    },
    "victory": {
      1: "切，我一定要找时间干掉你！"
    },
    "defeat": {
      1: "大老远跑来就这，昂？"
    }
  },
  "skull_boss_guzma_2": {
    "encounter": {
      1: `不管谁与我作对，我都会干掉他们！
          $这就是大坏蛋古兹马的全部！`
    },
    "victory": {
      1: "古兹马！！你在干什么啊！！！"
    },
    "defeat": {
      1: "你们真够蠢的!"
    }
  },
  "macro_boss_rose_1": {
    "encounter": {
      1: `我必须为所有人提供无限的能源，实现繁荣。
          $这是我的目的，我的责任，我的使命！`
    },
    "victory": {
      1: `你还是没有明白，训练师
          $我们…不，我要推动历史的进程啊！`
    },
    "defeat": {
      1: "你完全不理解！"
    }
  },
  "macro_boss_rose_2": {
    "encounter": {
      1: `我致力于解决伽勒尔的能源问题\n——当然也是全世界的能源问题。
          $我的经验与成果，造就了马洛科蒙集团，证明了我的正确与成功！
          $就算输了，我也不会改变主意的……`

    },
    "victory": {
      1: `我都忘记宝可梦对战的魅力了！我太久没有着手对战了…
          $这的确让人很满足，我接受我的失败！`
    },
    "defeat": {
      1: `我承认我做的事情非常渗人，我也不指望你能理解。
          $但我必须为伽勒尔地区提供无限的能源，确保永久的繁荣。`
    }
  },
```

40chyan commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -19,6 +19,12 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "银河队老大",
   "plasma_boss": "等离子队老大",
   "flare_boss": "闪焰队老大",
+  "aether_boss": "Aether President",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_boss": "Team Skull Boss",
+  "skull_admin": "Team Skull Admin",
+  "macro_boss": "Macro Cosmos President",
+  "macro_admin": "Macro Cosmos",
```
```suggestion
  "aether_boss": "以太基金会理事长",
  "aether_admin": "以太基金会干部",
  "skull_boss": "骷髅队老大",
  "skull_admin": "骷髅队干部",
  "macro_boss": "马洛科蒙总裁",
  "macro_admin": "马洛科蒙",
```

40chyan commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -137,6 +143,12 @@ export const trainerClasses: SimpleTranslationEntries = {
   "plasma_grunt_female": "等离子队手下",
   "flare_grunt": "闪焰队手下",
   "flare_grunt_female": "闪焰队手下",
+  "aether_grunt": "Aether Foundation Employee",
+  "aether_grunt_female": "Aether Foundation Employee",
+  "skull_grunt": "Team Skull Grunt",
+  "skull_grunt_female": "Team Skull Grunt",
+  "macro_grunt": "Macro Cosmos Trainer",
+  "macro_grunt_female": "Macro Cosmos Trainer",
```
```suggestion
  "aether_grunt": "以太基金会职员",
  "aether_grunt_female": "以太基金会职员",
  "skull_grunt": "骷髅队手下",
  "skull_grunt_female": "骷髅队手下",
  "macro_grunt": "马洛科蒙训练师",
  "macro_grunt_female": "马洛科蒙训练师",
```

40chyan commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -328,6 +340,13 @@ export const trainerNames: SimpleTranslationEntries = {
   "cyrus": "赤日",
   "ghetsis": "魁奇思",
   "lysandre": "弗拉达利",
+  "faba": "Faba",
+  "lusamine": "Lusamine",
+  "plumeria": "Plumeria",
+  "guzma": "Guzma",
+  "oleana": "Oleana",
+  "rose": "Rose",
```
```suggestion
  "faba": "扎奥博",
  "lusamine": "露莎米奈",
  "plumeria": "布尔美丽",
  "guzma": "古兹马",
  "oleana": "奥莉薇",
  "rose": "洛兹",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -19,6 +19,12 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "갤럭시단 보스",
   "plasma_boss": "플라스마단 보스",
   "flare_boss": "플레어단 보스",
+  "aether_boss": "Aether President",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_boss": "Team Skull Boss",
+  "skull_admin": "Team Skull Admin",
+  "macro_boss": "Macro Cosmos President",
+  "macro_admin": "Macro Cosmos",
```
```suggestion
  "aether_boss": "에테르재단 대표",
  "aether_admin": "에테르재단 지부장",
  "skull_boss": "스컬단 보스",
  "skull_admin": "스컬단 간부",
  "macro_boss": "매크로코스모스 사장",
  "macro_admin": "매크로코스모스 간부",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -137,6 +143,12 @@ export const trainerClasses: SimpleTranslationEntries = {
   "plasma_grunt_female": "플라스마단 조무래기",
   "flare_grunt": "플레어단 조무래기",
   "flare_grunt_female": "플레어단 조무래기",
+  "aether_grunt": "Aether Foundation Employee",
+  "aether_grunt_female": "Aether Foundation Employee",
+  "skull_grunt": "Team Skull Grunt",
+  "skull_grunt_female": "Team Skull Grunt",
+  "macro_grunt": "Macro Cosmos Trainer",
+  "macro_grunt_female": "Macro Cosmos Trainer",
```
```suggestion
  "aether_grunt": "에테르재단 직원",
  "aether_grunt_female": "에테르재단 직원",
  "skull_grunt": "스컬단 단원",
  "skull_grunt_female": "스컬단 단원",
  "macro_grunt": "매크로코스모스 사원",
  "macro_grunt_female": "매크로코스모스 사원",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -270,6 +282,12 @@ export const trainerNames: SimpleTranslationEntries = {
   "cyrus": "태홍",
   "ghetsis": "게치스",
   "lysandre": "플라드리",
+  "faba": "Faba",
+  "lusamine": "Lusamine",
+  "plumeria": "Plumeria",
+  "guzma": "Guzma",
+  "oleana": "Oleana",
+  "rose": "Rose",
```
```suggestion
  "faba": "자우보",
  "lusamine": "루자미네",
  "plumeria": "플루메리",
  "guzma": "구즈마",
  "oleana": "올리브",
  "rose": "로즈",
```

sodaMelon commented on src/locales/ko/bgm-name.ts:
```diff
@@ -79,11 +79,17 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_grunt": "BDSP 갤럭시단 배틀",
   "battle_plasma_grunt": "BW 플라스마단 배틀",
   "battle_flare_grunt": "XY 플레어단 배틀",
+  "battle_aether_grunt": "SM Aether Foundation Battle",
+  "battle_skull_grunt": "SM Team Skull Battle",
+  "battle_macro_grunt": "SWSH Trainer Battle",
```
```suggestion
  "battle_aether_grunt": "SM 에테르재단 배틀",
  "battle_skull_grunt": "SM 스컬단 배틀",
  "battle_macro_grunt": "SWSH Trainer Battle",
```

sodaMelon commented on src/locales/ko/bgm-name.ts:
```diff
@@ -79,11 +79,17 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_boss": "BDSP 태홍 배틀",
   "battle_plasma_boss": "B2W2 게치스 배틀",
   "battle_flare_boss": "XY 플라드리 배틀",
+  "battle_aether_boss":  "SM Lusamine Battle",
+  "battle_skull_boss": "SM Guzma Battle",
+  "battle_macro_boss": "SWSH Rose Battle",
```
```suggestion
  "battle_aether_boss":  "SM 루자미네 배틀",
  "battle_skull_boss": "SM 구즈마 배틀",
  "battle_macro_boss": "SWSH 로즈 배틀",
```

sodaMelon reviewed (commented):
added koren localization (about trainer, BGM)
I will do the other on next comment

EnochG1 commented:
@Blitz425 
From now Japanese is added in beta.
Could you add Japanese locale files when you merge latest beta(to resolve conflicts)? 🙏

Blitz425 commented:
> I like Bewear and I like Mega Lopunny but I'm not as big of a fan of just Lopunny. I also am not sure I like Nihilego right in the first team. If you wanted to keep the theme of boss1 having a mega then replacing Nihilego with Mega Lopunny would be perfect.

Since Nihilego is 10 bst under Mega Lopunny and she doesn't canonically use it as her mega, I figured it would be a better fit on that slot for fight 1. 

> Was Stakataka chosen as a variant because of typing? I would have personally gone for something like Naganadel (a bulky UB and a fast offensive UB provide different team roles but fulfill similar team thematic roles). Necrozma is....fine objectively speaking because it's just 600BST but somehow it seems to feel like it has more impact because it has more powerful forms. Did you end up ditching the idea of Flutter Mane and Scream Tail here? Are Milotic/Primarina/Clefable strong enough or thematic enough to justify carrying over 2 team slots from the first fight?

Stakataka fit the same typing / defensive niche that Celesteela does. I still need to test her team but she already has two fast offensive Pokemon in Pheromosa/Nihilego so I feel it would be overkill to introduce Naganadel like that. It would also make her team extremely susceptible to Psychic which is something I also took into consideration. I'm confused if you mean Necrozma is lackluster or replaceable? I did end up ditching the idea of Flutter / Scream since Ultra Beasts are more flavorful and they weren't providing much that others were already accomplishing. Milotic/Primarina are really strong both defensively and offensively, and I also didn't want the team to look like it was spamming sublegend BST mons. I consider Clefable her ace but Lusamine 2 has undergone like 5 reworks at this point so I'm open to ideas.

> Buzzwole not being a boss feels like a crime and would bump up the power level a lot. No reason for Goliospod to take its boss slot especially since it's not even a boss in the first fight. Not a fan of 1 segment bars because that just sticks out. Hisui Samurott also feels a little random on the team. In terms of power, I guarantee you that 3 bosses of Mega Pinsir, Xurkitree, and Buzzwole is on par with the other power levels.

It's a Weavile situation where it feels wrong not to give his ace boss bars but I get what you mean. I probably will end up just make the bosses Buzz/Xurk/Pinsir. Hisui Samurott is a bit random and was purely chosen based on theming / vibes / coverage. Seeing as other leaders get stuff like Great Tusk I thought it would be okay to take a bit of creative liberties in that case.

> Ghetsis 2: Replacing Durant with Genesect... it's fine I guess though I often see Genesect more associated with Colress. If you're going to be removing the paradox moths from Ghetsis you might as well also tweak his first team because that was part of the reason I had them on the second team. I also still like Darm/GDarm because they both hit very hard and have Unovan roots. My biggest complaint is that with your proposed changes there's only a single variation on his team now and that's Hydreigon/Iron Jugulis and removing so much variety is not good in my opinion

I'll maybe consider tweaking his first team but I felt Volc/Slither was okay because in every other evil team, you had stuff like Tyranitar/Iron Thorns, Salamence/Roaring Moon, etc. all on one slot (Slot that shared both the base form and its paradox version) and added bonus they provide the same general offensive / defensive capabilities. The Darms do indeed hit hard but the rest of their stats are really lackluster and their typing was already covered by Volc (previously Iron Moth) and Kyurem, rolling Galar Darm and a Hydreigon meant 4 out of 6 members on the team were just susceptible to fighting and couldn't really do much about it which was partially the reason why I gave him Basculegion for a fighting immunity. How would you feel about Basculegion/Jellicent on one slot? I personally consider Genesect a technical dual slot since I made it choose a random drive for added "randomness" but Jellicent would be incredibly good on him if you're worried about its performance.

CodeTappert commented on src/locales/de/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_grunt": "BDSP Vs. Team Galaktik Rüpel",
   "battle_plasma_grunt": "SW Vs. Team Plasma Rüpel",
   "battle_flare_grunt": "XY Vs. Team Flare Rüpel",
+  "battle_aether_grunt": "SM Vs. Æther Foundation",
+  "battle_skull_grunt": "SM Vs. Team Skull Rüpel",
+  "battle_macro_grunt": "SWSH Vs. Trainer",
+  "battle_galactic_admin": "BDSP Team Galactic Admin Battle",
+  "battle_skull_admin": "SM Team Skull Admin Battle",
+  "battle_oleana": "SWSH Oleana Battle",
```
```suggestion
  "battle_aether_grunt": "SM Vs. Æther Foundation",
  "battle_skull_grunt": "SM Vs. Team Skull Rüpel",
  "battle_macro_grunt": "SWSH Vs. Trainer",
  "battle_galactic_admin": "BDSP Vs. Team Galactic Commander",
  "battle_skull_admin": "SM Vs. Team Skull Vorstand",
  "battle_oleana": "SWSH Vs. Oleana",
```

CodeTappert commented on src/locales/de/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunt": "Rüpel von Team Flare",
   "flare_grunt_female": "Rüpel von Team Flare",
   "flare_grunts": "Rüpel von Team Flare",
+  "aether_grunt": "Æther Foundation Personal",
+  "aether_grunt_female": "Æther Foundation Personal",
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "Rüpel von Team Skull",
+  "skull_grunt_female": "Rüpel von Team Skull",
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "Angestellter von Macro Cosmos",
+  "macro_grunt_female": "Angestellte von Macro Cosmos",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "aether_grunt": "Æther Foundation Personal",
  "aether_grunt_female": "Æther Foundation Personal",
  "aether_grunts": "Æther Foundation Personal",
  "skull_grunt": "Rüpel von Team Skull",
  "skull_grunt_female": "Rüpel von Team Skull",
  "skull_grunts": "Rüpel von Team Skull",
  "macro_grunt": "Angestellter von Macro Cosmos",
  "macro_grunt_female": "Angestellte von Macro Cosmos",
  "macro_grunts": "Angestellte von Macro Cosmos",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -156,7 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
-
+  "aether_grunt": "에테르재단 직원",
+  "aether_grunt_female": "에테르재단 직원",
+  "aether_grunts": "Aether Foundation Employees",
```
```suggestion
  "aether_grunts": "에테르재단 직원들",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -156,7 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "스컬단 조무래기",
+  "skull_grunt_female": "스컬단 조무래기",
+  "skull_grunts": "Team Skull Grunts",
```
```suggestion
  "skull_grunts": "스컬단 조무래기들",
```

sodaMelon commented on src/locales/ko/trainers.ts:
```diff
@@ -156,7 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "매크로코스모스 직원",
+  "macro_grunt_female": "매크로코스모스 직원",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "macro_grunts": "매크로코스모스 직원들",
```

sodaMelon commented on src/locales/ko/dialogue.ts:
```diff
@@ -808,6 +888,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
+    "encounter": {
+      1: `Doesn't matter who I'm up against, I'm gonna beat them down!
+          $That's what big bad Guzma is all about!`
+    },
+    "victory": {
+      1: "Guzma!!! What is wrong with you?!"
+    },
+    "defeat": {
+      1: "Y'all are stupid!"
+    }
+  },
+  "macro_boss_rose_1": {
+    "encounter": {
+      1: `I must provide limitless energy to ensure everlasting prosperity for everyone...
+          $It is my purpose, my duty, my destiny!`
+    },
+    "victory": {
+      1: `You still don't understand, trainer...
+          $We... No, I am going to change the course of history!`
+    },
+    "defeat": {
+      1: "You still don't understand a thing!"
+    }
+  },
+  "macro_boss_rose_2": {
+    "encounter": {
+      1: `I'm committed to solving the energy problem in the Galar region—and, of course, around the world.
+          $My experience and accomplishments that made Macro Cosmos a success are proof my methods work.
+          $I don't intend to change my mind, even if I lose.`
+
+    },
+    "victory": {
+      1: `I'd forgotten how great Pokémon battles are! It's been so long since I battled...
+          $That sure was satisfying, I accept defeat for this battle.`
+    },
+    "defeat": {
+      1: `I suppose it must seem that I am doing something terrible. I don't expect you to understand.
+          $But I must provide the Galar region with limitless energy to ensure everlasting prosperity.`
+    }
+  },
```
```suggestion
  "aether_boss_lusamine_1": {
    "encounter": {
      1: "비스트가 놀라잖아! …우선은 너를 조용히 시켜야겠어."
    },
    "victory": {
      1: "..너무해!"
    },
    "defeat": {
      1: "흐음..."
    }
  },
  "aether_boss_lusamine_2": {
    "encounter": {
      1: `그렇게 말했는데도! ...왜 이리 끈질긴 거야! 정말 진절머리가 나.
          $더 말해봐야 소용 없겠지… 텅비드의 힘으로 네가 여기까지 온 걸 후회하게 해줄게!`
    },
    "victory": {
      1: "...아아아웃!!"
    },
    "defeat": {
      1: "나는 비스트만 있으면 돼! 너희가 어떻게 되든 상관없어!"
    }
  },
  "skull_boss_guzma_1": {
    "encounter": {
      1: `박살 내고 박살 내도 절대 봐주지 않아...
          $모두가 싫어하는 구즈마가 여기 있다!`
    },
    "victory": {
      1: "쯧. 너는 내가 박살 내주겠어!"
    },
    "defeat": {
      1: "겨우 그걸 위해서 일부러 온 거야?"
    }
  },
  "skull_boss_guzma_2": {
    "encounter": {
      1: `누구든지 박살 낸다!
          $그것이 구즈마님의 아이덴티티!!`
    },
    "victory": {
      1: "구즈마!!! 뭐 하고 있는 거야?!"
    },
    "defeat": {
      1: "너희들 바보군!"
    }
  },
  "macro_boss_rose_1": {
    "encounter": {
      1: `내게는 모든 사람이 영원히 마음을 놓고 발전할 수 있도록...
          $무한한 에너지를 가져다 줘야 한다는 신념과 사명이 있단다!`
    },
    "victory": {
      1: `아직 무르군… 트레이너.
          $역사는 바뀔 것이네! 이 나의 손에 의해서 말이지`
    },
    "defeat": {
      1: "자네는 여전히 깨닫지 못했군!"
    }
  },
  "macro_boss_rose_2": {
    "encounter": {
      1: `나는 가라르를 포함한— 전 세계의 에너지 문제를 해결하기 위해 최선을 다하고 있어.
          $매크로코스모스를 성공으로 이끈 경험과 성과는 내 방식이 효율적이라는 증거란다.
          $생각을 바꿀 마음은 없어. 내가 지더라도 말이네.`

    },
    "victory": {
      1: `포켓몬 승부는 좋군! 오랜만에 싸워보니 만족스러워...
          $제법 괜찮군. 패배를 인정하지.`
    },
    "defeat": {
      1: `너희가 보기에는 내가 끔찍한 짓을 벌이고 있는 것처럼 보이겠지? 조금도 이해가 가지 않을 거야.
          $하지만 난 가라르지방의 영원한 번영을 위해서 무한한 에너지를 가져다줘야 해.`
    }
  },
```

CodeTappert commented on src/locales/de/dialogue.ts:
```diff
@@ -839,6 +919,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
+    "encounter": {
+      1: `Doesn't matter who I'm up against, I'm gonna beat them down!
+          $That's what big bad Guzma is all about!`
+    },
+    "victory": {
+      1: "Guzma!!! What is wrong with you?!"
+    },
+    "defeat": {
+      1: "Y'all are stupid!"
+    }
+  },
+  "macro_boss_rose_1": {
+    "encounter": {
+      1: `I must provide limitless energy to ensure everlasting prosperity for everyone...
+          $It is my purpose, my duty, my destiny!`
+    },
+    "victory": {
+      1: `You still don't understand, trainer...
+          $We... No, I am going to change the course of history!`
+    },
+    "defeat": {
+      1: "You still don't understand a thing!"
+    }
+  },
+  "macro_boss_rose_2": {
+    "encounter": {
+      1: `I'm committed to solving the energy problem in the Galar region—and, of course, around the world.
+          $My experience and accomplishments that made Macro Cosmos a success are proof my methods work.
+          $I don't intend to change my mind, even if I lose.`
+
+    },
+    "victory": {
+      1: `I'd forgotten how great Pokémon battles are! It's been so long since I battled...
+          $That sure was satisfying, I accept defeat for this battle.`
+    },
+    "defeat": {
+      1: `I suppose it must seem that I am doing something terrible. I don't expect you to understand.
+          $But I must provide the Galar region with limitless energy to ensure everlasting prosperity.`
+    }
+  },
```
```suggestion
  "aether_boss_lusamine_1": {
    "encounter": {
      1: `Du wirst mein süßes Biest erschrecken! Wie kannst du nur? Ultra-Besiten sind so niedlich!
          $Es sieht so aus, als müsste ich dich zuerst zum Schweigen bringen.`
    },
    "victory": {
      1: "Wie kann eine Person nur so grausam sein?!"
    },
    "defeat": {
      1: "Und jetzt, mein süßes Biest, komm zu Mama!"
    }
  },
  "aether_boss_lusamine_2": {
    "encounter": {
      1: `Wieso musst du mich weiterhin belästigen?! Ich habe die Nase voll von dir.
         $Du machst mich krank! Genug mit diesem nutzlosen Gerede... 
         $Mit der Kraft von Anego, von uns als UB-01 Schmarotzer klassifiziert,
         $werde ich dir zeigen, wie falsch du warst, hierher zu kommen!`
    },
    "victory": {
      1: "Aaauuuggghhhhhhhhh!!!"
    },
    "defeat": {
      1: "Alles, was ich will, ist mein kostbares Biest! Der Rest von euch ist mir egal!"
    }
  },
  "skull_boss_guzma_1": {
    "encounter": {
      1: `Kein Erbarmen! Keine Gnade! Ich bin dein schlimmster Alptraum! 
         $Oh yeah, ich bin der große, böse Bromley!`
    },
    "victory": {
      1: "Irgendwann mach ich dich platt! Ich hab noch ein Ass im Ärmel. Du wirst schon sehen."
    },
    "defeat": {
      1: "Du hast den langen Weg hierher DAFÜR aufgenommen? Pah!"
    }
  },
  "skull_boss_guzma_2": {
    "encounter": {
      1: "Mir ist egal, wer du bist. Ich mach dich platt!! Das schreibt mir mein Ego vor. Ich bin Bromley!"
    },
    "victory": {
      1: "Bromley!!! Was ist falsch mit dir?!"
    },
    "defeat": {
      1: "Ihr seid wirklich selten dämlich."
    }
  },
  "macro_boss_rose_1": {
    "encounter": {
      1: `Mein Name ist Rose. Ich bin der Präsident der Galar Pokémon-Liga und Geschäftsführer von Macro Cosmos
          $Ich habe eine Aufgabe, eine Berufung! Ich muss die Galar-Region mit unbegrenzter Energie versorgen,
          damit sie sich bis in alle Ewigkeit in Frieden fortentwickeln kann!`
    },
    "victory": {
      1: "Du versteht es einfach nicht...So können wir die Geschichte nicht verändern!"
    },
    "defeat": {
      1: "Ohne deine Einmischung kann ich jetzt endlich weiterarbeiten!"
    }
  },
  "macro_boss_rose_2": {
    "encounter": {
      1: `Ich bin fest entschlossen, das Energieproblem in der Galar-Region
         $und natürlich auf der ganzen Welt zu lösen.
         $Meine Erfahrungen und Erfolge, die Macro Cosmos zu einem Erfolg gemacht haben,
         $sind der Beweis, dass meine Methoden funktionieren.
         $Ich habe nicht vor, meine Meinung zu ändern, selbst wenn ich verliere.`

    },
    "victory": {
      1: `Ich hatte vergessen, wie großartig Pokémon-Kämpfe sind! Es ist so lange her, dass ich gekämpft habe...
          $Das war wirklich befriedigend, ich akzeptiere die Niederlage für diesen Kampf.`
    },
    "defeat": {
      1: `Ich nehme an, es muss den Anschein haben, dass ich etwas Schreckliches tue.
          $Ich erwarte nicht, dass du es verstehst. Aber ich muss der Galar-Region grenzenlose Energie
          $bereitstellen, um ewigen Wohlstand zu gewährleisten.`
    }
  },
```

CodeTappert commented on src/locales/de/dialogue.ts:
```diff
@@ -696,6 +696,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      4: "Wha-?! This kid's way too strong-no bones about it!",
+      5: "So, what? I'm lower than a Pokémon?! I already got self-esteem issues, man."
+    },
+  },
+  "plumeria": {
+    "encounter": {
+      1: " ...Hmph. You don't look like anything special to me.",
+      2: "It takes these dumb Grunts way too long to deal with you kids..",
+      3: "Mess with anyone in Team Skull, and I'll show you how serious I can get."
+    },
+    "victory": {
+      1: "Hmmph! You're pretty strong. I'll give you that.",
+      2: "Hmmph. Guess you are pretty tough. Now I understand why my Grunts waste so much time battling kids.",
+      3: "Hmmph! I guess I just have to hold that loss."
+    },
+  },
+  "macro_grunt": {
+    "encounter": {
+      1: "It looks like this is the end of the line for you!",
+      2: "You are a trainer aren't you? I'm afraid that doesn't give you the right to interfere in our work.",
+      3: "I'm from Macro Cosmos Insurance! Do you have a life insurance policy?"
+    },
+    "victory": {
+      1: "I have little choice but to respectfully retreat.",
+      2: "Having to give up my pocket money... Losing means I'm back in the red...",
+      3: "Nobody can beat Macro Cosmos when it comes to our dedication to our work!"
+    },
+  },
+  "oleana": {
+    "encounter": {
+      1: "I won't let anyone interfere with Mr. Rose's plan!",
+      2: "So, you got through all of the special staff that I had ordered to stop you. I would expect nothing less.",
+      3: "For the chairman! I won't lose!"
+    },
+    "victory": {
+      1: "*sigh* I wasn't able to win... Oleana...you really are a hopeless woman.",
+      2: "Arghhh! This is inexcusable... What was I thinking... Any trainer who's made it this far would be no pushover..",
+      3: "*sigh* I am one tired Oleana..."
+    },
+  },
```
```suggestion
  "aether_grunt": {
    "encounter": {
      1: "Ich werde dich mit allem, was ich habe, bekämpfen. Ich werde dich auslöschen!",
      2: `Mir ist egal, ob du ein simpler Trainer bist oder nicht.
          $Ich werde dich hochkant rauswerfen, wenn du uns bedrohst!`,
      3: "Mir wurde gesagt, Trainer abzuweisen, egal wer sie sind!",
      4: "Ich zeige dir die Macht des Æther-Paradies",
      5: "Jetzt da du die Wahre Natur des Æther-Paradies kennst, müssen wir dich leider beseitigen!"
    },
    "victory": {
      1: "Hmph! Du scheinst ziemlich geschickt zu sein.",
      2: "Was hat das zu bedeuten? Was hat das zu bedeuten!",
      3: "Hey, du bist so stark, dass ich dich nicht abweisen kann!",
      4: "Hmm... Es scheint, als hätte ich verloren.",
      5: "Ich hätte das Blitzdings benutzen sollen..."
    },
  },
  "faba": {
    "encounter": {
      1: "Ich bin Regionalleiter Fabian und ich werde dir zeigen, wie hart die Welt sein kann!",
      2: "Die letzte Verteidigungslinie des Æther-Paradieses kämpft gegen einen simplen Trainer?",
      3: `Ich Fabian, bin der Regionalleiter der Æther Foundation.
          $Der einzige auf der Welt, ich bin unersetzlich.`
    },
    "victory": {
      1: "Aiyee!",
      2: "W-w-wie kann das sein?! Wie konnte dieser Trainer...",
      3: "Das ist der Grund... warum ich andere Menschen nicht ausstehen kann."
    },
  },
  "skull_grunt": {
    "encounter": {
      1: "Wir sind nicht böse, wir sind einfach nur nicht nett!",
      2: "Willst du Stress, oder was? Das ist unsere Art, Hallo zu sagen! Tschüss, du Punk!",
      3: "Wir sind nur ein paar Typen und Mädels mit großem Interesse an den Pokémon anderer Leute!",
      4: "Wieso machst du einen auf hart, wenn wir schon hart wie Knochen sind, Brudi?",
      5: `Yo, platz hier nich’ rein, ♪ wie’n wildes Kleinstein! ♪ Wenn du meinst, dass Team Skull ♪
         $an nix denkt als Krawall, ♪ hast du leider ’nen Knall, ♪ denn das is’ nich’ der Fall! ♪
         $Wir woll’n hier nur gammeln ♪ und gechillt Staub ansammeln! ♪`
    },
    "victory": {
      1: "Wie jetzt? Schon vorbei?",
      2: "Woah, lass uns lieber die Kurve kratzen, yo! Kein Bock, hier meinen Schädel zu riskieren...",
      3: "Wir brauchen deine schwachen Pokémon eh nicht!",
      4: "Was? Deine Stärke spür ich bis in die Knochen...",
      5: "Mein Homie und ich sind also weniger wert als Pokémon? Willste uns dissen, oder was?"
    },
  },
  "plumeria": {
    "encounter": {
      1: "...Hmph. Du siehst nicht so aus, als wärst du etwas Besonderes.",
      2: "Die blöden Rüpel brauchen viel zu lange, um mit euch gewöhnlichen Trainern fertig zu werden...",
      3: "Lege dich mit irgendjemandem in Team Skull an, und ich zeige dir, wie ernst ich werden kann."
    },
    "victory": {
      1: "Du bist ziemlich stark. Das muss ich zugeben.",
      2: "Jetzt versteh ich, warum meine Rüpel so viel Zeit damit verschwenden...",
      3: "Ich denke, ich muss diese Niederlage einfach hinnehmen."
    },
  },
  "macro_grunt": {
    "encounter": {
      1: "Achtung hier ist Endstation für dich!",
      2: `Du bist ein Trainer, oder? Wir von MC Wertpapiere wissen so etwas. 
          $Ich fürchte, das gibt dir trotzdem nicht das Recht, sich in unsere Arbeit einzumischen.`,
      3: "Ich bin von MC Versicherungen! Hast du eine Lebensversicherung?"
    },
    "victory": {
      1: "Ich habe keine andere Wahl, als respektvoll zurückzutreten.",
      2: "Mein Erspartes aufzugeben bringt mich in die roten Zahlen...",
      3: "Okay zurück an die Arbeit. Versicherungen verkauft sich nicht von alleine."
    },
  },
  "oleana": {
    "encounter": {
      1: "Ich werde keine Störung von Präsident Roses Plänen zulassen!",
      2: `Du hast es also durch die Mitarbeiter der Tochterunternehmen geschafft,
         $die ich beauftragt habe, um dich aufzuhalten. Ich hätte nichts anderes erwartet.`,
      3: `Für den Präsidenten! Ich werde nicht verlieren! 
         $Mach dich bereit mit Macro Cosmos Airlines nach Hause zu fliegen!`
    },
    "victory": {
      1: "Ich habe es nicht geschafft... Olivia... du bist wirklich eine hoffnungslose Frau.",
      2: `Arghhh! Das ist unentschuldbar... Was habe ich mir dabei gedacht...
          $Jeder Trainer, der es so weit geschafft hat, muss ein harter Brocken sein...`,
      3: "Ich bin eine müde Olivia... Ob es Macro Cosmos Betten gibt?"
    },
  },
```

sodaMelon commented on src/locales/ko/dialogue.ts:
```diff
@@ -676,6 +676,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      4: "Wha-?! This kid's way too strong-no bones about it!",
+      5: "So, what? I'm lower than a Pokémon?! I already got self-esteem issues, man."
+    },
+  },
+  "plumeria": {
+    "encounter": {
+      1: " ...Hmph. You don't look like anything special to me.",
+      2: "It takes these dumb Grunts way too long to deal with you kids..",
+      3: "Mess with anyone in Team Skull, and I'll show you how serious I can get."
+    },
+    "victory": {
+      1: "Hmmph! You're pretty strong. I'll give you that.",
+      2: "Hmmph. Guess you are pretty tough. Now I understand why my Grunts waste so much time battling kids.",
+      3: "Hmmph! I guess I just have to hold that loss."
+    },
+  },
+  "macro_grunt": {
+    "encounter": {
+      1: "It looks like this is the end of the line for you!",
+      2: "You are a trainer aren't you? I'm afraid that doesn't give you the right to interfere in our work.",
+      3: "I'm from Macro Cosmos Insurance! Do you have a life insurance policy?"
+    },
+    "victory": {
+      1: "I have little choice but to respectfully retreat.",
+      2: "Having to give up my pocket money... Losing means I'm back in the red...",
+      3: "Nobody can beat Macro Cosmos when it comes to our dedication to our work!"
+    },
+  },
+  "oleana": {
+    "encounter": {
+      1: "I won't let anyone interfere with Mr. Rose's plan!",
+      2: "So, you got through all of the special staff that I had ordered to stop you. I would expect nothing less.",
+      3: "For the chairman! I won't lose!"
+    },
+    "victory": {
+      1: "*sigh* I wasn't able to win... Oleana...you really are a hopeless woman.",
+      2: "Arghhh! This is inexcusable... What was I thinking... Any trainer who's made it this far would be no pushover..",
+      3: "*sigh* I am one tired Oleana..."
+    },
+  },
```
```suggestion
    "encounter": {
      1: "죽을 각오로 퇴치하겠습니다!",
      2: "소중한 연구라고! 애라도 가만두지 않겠어!",
      3: "어떤 트레이너든 물리치라고 들었다고!",
      4: "에테르파라다이스의 파워를 보여드리겠습니다!",
      5: "에테르파라다이스 중심의 어둠을 알아내다니. 이제 사라져 주셔야겠습니다!"
    },
    "victory": {
      1: "흥! 실력이 대단하시군요.",
      2: "이게 뭐하는 상황이지? 무슨 결과냐고!",
      3: "이런! 넌 너무 강해서 물리칠 방법도 없다고!",
      4: "으음… 제가 진 것처럼 보이네요.",
      5: "이 느낌을 한마디로 하면. 게에이~힝!!"
    },
  },
  "faba": {
    "encounter": {
      1: "나, 에테르재단의 자우보! 꼬맹이 녀석에게 현실을 보여주겠어요!",
      2: "에테르파라다이스 마지막 보루라고 불리는, 이 자우보에게 애를 상대하라고요?",
      3: "에테르재단의 지부장이라면 세계에 단 한 명… 이 자우보뿐입니다."
    },
    "victory": {
      1: "게에이~힝!!",
      2: "뭐 뭐뭐뭐 뭐가 어떻게 된 거죠? 내가 애를 상대로…",
      3: "이래서 애는… 좋아할 수가 없어요."
    },
  },
  "skull_grunt": {
    "encounter": {
      1: "나쁜 짓 안해-그냥 사는 게 힘들 뿐스컬!",
      2: "껌 하나 드실? 이게 우리가 인사하는 방식YO! 만나서 반갑스컬!",
      3: "우린 그냥 다른 사람의 포켓몬에 관심이 많을 뿐임다!",
      4: "어이 어이! 잘난 체하는 녀석한테 무시당하고 있잖스컬!!",
      5: "우리는 리프리젠트 땡땡이 스컬-! 집세는 못 내겠지만-! 정말 즐거웠지만, 청춘을 날려YO!"
    },
    "victory": {
      1: "엥? 벌써 끝났슴까?",
      2: "미는 엑시트! 유는 피쓰풀!",
      3: "꼬마 녀석의 이상한 포켓몬엔 관심 없어!",
      4: "뭐야-?! 너무 강해서 뼈도 못 추리겠어!",
      5: "우리가 포켓몬보다 아래야!? 소중히 대해주면 안 돼? 으응!?"
    },
  },
  "plumeria": {
    "encounter": {
      1: "…아무것도 느껴지지 않는 평~범한 애로 보이는데!",
      2: "워낙 애들 상대로 우물쭈물 하길래 오래 걸릴 줄 알았지…",
      3: "누구라도 스컬단 또 방해하면, 정말로 혼내줄 테니까!"
    },
    "victory": {
      1: "흥! 대단하군.",
      2: "흐음, 너 참 대단하군. 애들을 상대로 질질 끈 이유가 이해될 정도로 강해.",
      3: "흠. 이 손실은 그냥 안고 가야겠어."
    },
  },
  "macro_grunt": {
    "encounter": {
      1: "당신은 여기서 끝날 것 같네요!",
      2: "당신은 트레이너 맞죠? 하지만 우리를 방해하는 건 용납 못 합니다!",
      3: "매크로코스모스 생명입니다! 가입하신 실비보험은 있으신가요?"
    },
    "victory": {
      1: "순순히 물러나는 것 말고는 선택지가 없군요.",
      2: "용돈을 뺏기다니… 패배는 적자로 이어지는구나…",
      3: "매크로코스모스 생명에 관한 일이라면 누구에게도 지지 않을 텐데…"
    },
  },
  "oleana": {
    "encounter": {
      1: "로즈님을 방해하는 것은 절대로 용서할 수 없으니까요!",
      2: "저의 지시를 따르는 특별한 스태프들을 아무렇지도 않게 쓰러뜨리다니. …더 기대되는 것도 없네요.",
      3: "로즈님을 위해서라면! 질 수 없습니다!"
    },
    "victory": {
      1: "아아… 이기지 못하다니… 올리브… 넌 정말 구제 불능이구나.",
      2: "하아아…! 이런 실수를 하다니… 승리를 거듭한 트레이너의 실력을 너무 얕잡아봤던 모양이네요.",
      3: "아아… 이 올리브님 조금 지쳤어…"
    },
  },
```

sodaMelon reviewed (commented):
Last comment in Korean localization

Adri1 commented on src/locales/fr/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunts": "Sbires de la Team Flare",
+  "aether_grunt": "Employé de la Fondation Æther",
+  "aether_grunt_female": "Employée de la Fondation Æther",
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "Sbire de la Team Skull",
+  "skull_grunt_female": "Sbire de la Team Skull",
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "Employé de Macro Cosmos",
+  "macro_grunt_female": "Employée de Macro Cosmos",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "aether_grunts": "Employés de la Fondation Æther",
  "skull_grunt": "Sbire de la Team Skull",
  "skull_grunt_female": "Sbire de la Team Skull",
  "skull_grunts": "Team Skull Grunts",
  "macro_grunt": "Employé de Macro Cosmos",
  "macro_grunt_female": "Employée de Macro Cosmos",
  "macro_grunts": "Employés de Macro Cosmos",
```

Blitz425 commented:
Changed a couple things
- Ghetsis 2 now has Jellicent on Basculegion's slot for more team variety as he was lacking in that department.
- Lysandre 1 now has Dragalge / Clawitzer over Milotic to give to Lusamine. (Temp mentioned wanting to go back and give more Leaders variety and Lysandre's first team was severly lacking compared to any other leader). Changed Heliolisk for Malamar as it provides to the team better than Heliolisk and fits as a thematic 'evil' Pokemon from his generation. (Not too sure about this change but I feel Dragalge / Clawitzer are a good fit). Pyroar was made guaranteed male for the hair and theme 🗿
- Guzma 2 lost his one bar boss, Crawdaunt is now on Hisui Samurott's slot for more variety. Mega Pinsir is no longer the boss replaced with Buzzwole.

Adri1 commented on src/locales/fr/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_grunt": "DÉPS Vs. Team Galaxie",
   "battle_plasma_grunt": "NB - Vs. Team Plasma",
   "battle_flare_grunt": "XY - Vs. Team Flare",
+  "battle_aether_grunt": "SM Aether Foundation Battle",
+  "battle_skull_grunt": "SM Team Skull Battle",
+  "battle_macro_grunt": "SWSH Trainer Battle",
+  "battle_galactic_admin": "BDSP Team Galactic Admin Battle",
+  "battle_skull_admin": "SM Team Skull Admin Battle",
+  "battle_oleana": "SWSH Oleana Battle",
   "battle_rocket_boss": "USUL - Vs. Giovanni",
   "battle_aqua_magma_boss": "ROSA - Vs. Max/Arthur",
   "battle_galactic_boss": "DÉPS - Vs. Hélio",
   "battle_plasma_boss": "N2B2 - Vs. Ghetis",
   "battle_flare_boss": "XY - Vs. Lysandre",
+  "battle_aether_boss":  "SM Lusamine Battle",
+  "battle_skull_boss": "SM Guzma Battle",
+  "battle_macro_boss": "SWSH Rose Battle",
```
```suggestion
  "battle_aether_grunt": "SL - Vs. Fondation Æther",
  "battle_skull_grunt": "SL - Vs. Team Skull",
  "battle_macro_grunt": "ÉB - Vs. Macro Cosmos",
  "battle_galactic_admin": "DÉPS - Vs. Admin Team Galaxie",
  "battle_skull_admin": "SL - Vs. Admin Team Skull",
  "battle_oleana": "ÉB - Vs. Liv",
  "battle_rocket_boss": "USUL - Vs. Giovanni",
  "battle_aqua_magma_boss": "ROSA - Vs. Max/Arthur",
  "battle_galactic_boss": "DÉPS - Vs. Hélio",
  "battle_plasma_boss": "N2B2 - Vs. Ghetis",
  "battle_flare_boss": "XY - Vs. Lysandre",
  "battle_aether_boss":  "SL - Vs. Elsa-Mina",
  "battle_skull_boss": "SL - Vs. Guzma",
  "battle_macro_boss": "ÉB - Vs. Shehroz",
```

josericardo-fo commented on src/locales/pt_BR/trainers.ts:
```diff
@@ -19,6 +19,9 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Chefe da Equipe Galáctica",
   "plasma_boss": "Chefe da Equipe Plasma",
   "flare_boss": "Chefe da Equipe Flare",
+  "aether_boss": "Aether President",
+  "skull_boss": "Team Skull Boss",
+  "macro_boss": "Macro Cosmos President",
```
```suggestion
  "aether_boss": "Presidente Aether",
  "skull_boss": "Chefe da Equipe Skull",
  "macro_boss": "Presidente da Macro Cosmos",
```

josericardo-fo commented on src/locales/pt_BR/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunt": "Recruta da Equipe Flare",
   "flare_grunt_female": "Recruta da Equipe Flare",
   "flare_grunts": "Recrutas da Equipe Flare",
+  "aether_grunt": "Aether Foundation Employee",
+  "aether_grunt_female": "Aether Foundation Employee",
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "Team Skull Grunt",
+  "skull_grunt_female": "Team Skull Grunt",
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "Macro Cosmos Trainer",
+  "macro_grunt_female": "Macro Cosmos Trainer",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "aether_grunt": "Funcionário da Fundação Aether",
  "aether_grunt_female": "Funcionária da Fundação Aether",
  "aether_grunts": "Funcionários da Fundação Aether",
  "skull_grunt": "Capanga da Equipe Skull",
  "skull_grunt_female": "Capanga da Equipe Skull",
  "skull_grunts": "Capangas da Equipe Skull",
  "macro_grunt": "Treinador da Macro Cosmos",
  "macro_grunt_female": "Treinadora da Macro Cosmos",
  "macro_grunts": "Treinadores da Macro Cosmos",
```

josericardo-fo commented on src/locales/pt_BR/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_grunt": "BDSP Batalha da Equipe Galáctica",
   "battle_plasma_grunt": "BW Batalha da Equipe Plasma",
   "battle_flare_grunt": "XY Batalha da Equipe Flare",
+  "battle_aether_grunt": "SM Aether Foundation Battle",
+  "battle_skull_grunt": "SM Team Skull Battle",
+  "battle_macro_grunt": "SWSH Trainer Battle",
+  "battle_galactic_admin": "BDSP Team Galactic Admin Battle",
+  "battle_skull_admin": "SM Team Skull Admin Battle",
```
```suggestion
  "battle_aether_grunt": "SM Batalha da Fundação Aether",
  "battle_skull_grunt": "SM Batalha da Equipe Skull",
  "battle_macro_grunt": "SWSH Batalha de Treinador",
  "battle_galactic_admin": "BDSP Batalha com Admin da Equipe Galáctica",
  "battle_skull_admin": "SM Batalha com Admin da Euipe Skull",
```

josericardo-fo commented on src/locales/pt_BR/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_galactic_boss": "BDSP Batalha do Cyrus",
   "battle_plasma_boss": "B2W2 Batalha do Ghetsis",
   "battle_flare_boss": "XY Batalha do Lysandre",
+  "battle_aether_boss":  "SM Lusamine Battle",
+  "battle_skull_boss": "SM Guzma Battle",
+  "battle_macro_boss": "SWSH Rose Battle",
```
```suggestion
  "battle_aether_boss":  "SM Batalha da Lusamine",
  "battle_skull_boss": "SM Batalha do Guzma",
  "battle_macro_boss": "SWSH Batalha do Rose",
```

CodeTappert commented on src/battle-scene.ts:
```diff
@@ -1953,6 +1953,18 @@ export default class BattleScene extends SceneBase {
       return 12.974;
     case "battle_flare_grunt": //XY Team Flare Battle
       return 4.228;
+    case "battle_aether_grunt": // Battle!(Aether Foundation)
+      return 16.00;
+    case "battle_skull_grunt": // Battle!(Team Skull)
+      return 20.87;
+    case "battle_macro_grunt": // Battle!(Trainer) Sword / Shield
+      return 11.56;
```
Please keep the format of the comments same as every other. (Start with the abbreviation of the games)

CodeTappert commented on src/battle-scene.ts:
```diff
@@ -1963,6 +1975,12 @@ export default class BattleScene extends SceneBase {
       return 25.624;
     case "battle_flare_boss": //XY Lysandre Battle
       return 8.085;
+    case "battle_aether_boss": //Battle!(Lusamine)
+      return 11.33;
+    case "battle_skull_boss": //Battle!(Team Skull Boss)
+      return 13.13;
+    case "battle_macro_boss": //Battle!(Rose)
+      return 11.42;
```
Please keep the format of the comments same as every other. (Start with the abbreviation of the games)

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1809,32 +1863,40 @@ export const trainerConfigs: TrainerConfigs = {
-  [TrainerType.GHETSIS_2]: new TrainerConfig(++t).setName("Ghetsis").initForEvilTeamLeader("Plasma Boss", [], true).setMixedBattleBgm("battle_plasma_boss").setVictoryBgm("victory_team_plasma")
-    .setPartyMemberFunc(0, getRandomPartyMemberFunc([Species.SLITHER_WING, Species.IRON_MOTH], TrainerSlot.TRAINER, true, p => {
+  [TrainerType.GHETSIS_2]: new TrainerConfig(++t).setName("Ghetsis").initForEvilTeamLeader("Plasma Boss",[], true).setMixedBattleBgm("battle_plasma_boss").setVictoryBgm("victory_team_plasma")
+    .setPartyMemberFunc(0, getRandomPartyMemberFunc([ Species.HYDREIGON, Species.IRON_JUGULIS ], TrainerSlot.TRAINER, true, p => {
```
It's important to keep in mind that the first slot is actually the lowest level slot for bosses. The higher level spots are more in the back. Ghetsis also doesn't really lead with Hydreigon so I'd swap this with Volc/Slither

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1809,32 +1863,40 @@ export const trainerConfigs: TrainerConfigs = {
-    .setPartyMemberFunc(3, getRandomPartyMemberFunc([Species.KINGAMBIT]))
-    .setPartyMemberFunc(4, getRandomPartyMemberFunc([Species.HYDREIGON, Species.IRON_JUGULIS], TrainerSlot.TRAINER, true, p => {
+    .setPartyMemberFunc(1, getRandomPartyMemberFunc([ Species.BASCULEGION, Species.JELLICENT ], TrainerSlot.TRAINER, true, p => {
+      p.generateAndPopulateMoveset;
```
You're not calling the function here because you forgot the ()

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1809,32 +1863,40 @@ export const trainerConfigs: TrainerConfigs = {
+    .setPartyMemberFunc(0, getRandomPartyMemberFunc([ Species.MIENSHAO ]))
+    .setPartyMemberFunc(1, getRandomPartyMemberFunc([ Species.HONCHKROW, Species.TALONFLAME ]))
+    .setPartyMemberFunc(2, getRandomPartyMemberFunc([ Species.PYROAR ], TrainerSlot.TRAINER, true, p => {
+      p.generateAndPopulateMoveset;
```
Same issue here with not calling the function

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1809,32 +1863,40 @@ export const trainerConfigs: TrainerConfigs = {
+      p.generateAndPopulateMoveset;
+      p.gender = Gender.MALE;
+    }))
+    .setPartyMemberFunc(3, getRandomPartyMemberFunc([ Species.CLAWITZER, Species.DRAGALGE ]))
+    .setPartyMemberFunc(4, getRandomPartyMemberFunc([ Species.MALAMAR ]))
```
I kinda dig this. Not sure if I'm mandela'ing myself or what but I really do feel like Heliolisk works on him. Maybe as an alternative to Malamar for more type diversity?

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1845,12 +1907,15 @@ export const trainerConfigs: TrainerConfigs = {
       p.setBoss(true, 2);
       p.generateAndPopulateMoveset();
-      p.pokeball = PokeballType.ULTRA_BALL;
+      p.pokeball = PokeballType.MASTER_BALL;
```
I'd say master balls are reserved for the cover or really important legendaries. Everything else can stay ultra

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1845,12 +1907,15 @@ export const trainerConfigs: TrainerConfigs = {
-    .setPartyMemberFunc(3, getRandomPartyMemberFunc([Species.GOODRA, Species.HISUI_GOODRA]))
-    .setPartyMemberFunc(4, getRandomPartyMemberFunc([Species.GYARADOS], TrainerSlot.TRAINER, true, p => {
+    .setPartyMemberFunc(1, getRandomPartyMemberFunc([ Species.PYROAR ], TrainerSlot.TRAINER, true, p => {
+      p.generateAndPopulateMoveset;
```
another uncalled function

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1862,4 +1927,114 @@ export const trainerConfigs: TrainerConfigs = {
+    .setPartyMemberFunc(1, getRandomPartyMemberFunc([ Species.LILLIGANT, Species.HISUI_LILLIGANT ]))
+    .setPartyMemberFunc(2, getRandomPartyMemberFunc([ Species.CLEFABLE ], TrainerSlot.TRAINER, true, p => {
+	  p.generateAndPopulateMoveset;
+	  p.abilityIndex = 2; //Unaware
```
This feels like a weird change since it's not needed for the team (like how swift swim/chlorophyll is on Archie/Maxie) and the comment is also incorrect (Clefable's 2 ability is magic guard). I'd say to steer away from setting abilities/moves unless it's pivotal for the Pokemon or team. The generateAndPopulateMoveset is also not being called

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1862,4 +1927,114 @@ export const trainerConfigs: TrainerConfigs = {
+      p.generateAndPopulateMoveset();
+    })),
+  [TrainerType.LUSAMINE_2]: new TrainerConfig(++t).setName("Lusamine").initForEvilTeamLeader("Aether Boss",[], true).setMixedBattleBgm("battle_aether_boss").setVictoryBgm("victory_team_plasma")
+    .setPartyMemberFunc(0, getRandomPartyMemberFunc([ Species.NIHILEGO ], TrainerSlot.TRAINER, true, p => {
```
Same issue here with the first mon being the weakest. I originally purposefully made it so each boss2 had the bosses in the first, fifth, and sixth slots so it'd make their weakest and 2 strongest into bosses but you can probably just move this to a later slot

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1862,4 +1927,114 @@ export const trainerConfigs: TrainerConfigs = {
+      p.setBoss(true, 2);
+      p.generateAndPopulateMoveset();
+    }))
+    .setPartyMemberFunc(1, getRandomPartyMemberFunc([ Species.MILOTIC, Species.PRIMARINA ]))
+    .setPartyMemberFunc(2, getRandomPartyMemberFunc([ Species.CLEFABLE ], TrainerSlot.TRAINER, true, p => {
+	  p.generateAndPopulateMoveset;
+	  p.abilityIndex = 2; //Unaware
+    }))
```
Also okay with swapping one or both of these out for other Pokemon since right now she's carrying over 3 slots

CodeTappert replied:
![image](https://github.com/user-attachments/assets/a2c71d5b-cedb-41e8-b5d8-49e1533c2a0e)

She is doing this in the game also

CodeTappert replied:
![image](https://github.com/user-attachments/assets/fe233114-4dbb-48e9-8249-4f1acfb28573)

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1862,4 +1927,114 @@ export const trainerConfigs: TrainerConfigs = {
+    })),
+  [TrainerType.GUZMA]: new TrainerConfig(++t).setName("Guzma").initForEvilTeamLeader("Skull Boss",[]).setMixedBattleBgm("battle_skull_boss").setVictoryBgm("victory_team_plasma")
+    .setPartyMemberFunc(0, getRandomPartyMemberFunc([ Species.LOKIX, Species.YANMEGA ], TrainerSlot.TRAINER, true, p => {
+	  p.generateAndPopulateMoveset;
```
anotha one

Tempo-anon commented on src/data/trainer-config.ts:
```diff
@@ -1862,4 +1927,114 @@ export const trainerConfigs: TrainerConfigs = {
+    .setPartyMemberFunc(0, getRandomPartyMemberFunc([ Species.GOLISOPOD ], TrainerSlot.TRAINER, true, p => {
+      p.setBoss(true, 2);
+      p.generateAndPopulateMoveset();
+	  p.abilityIndex = 1; //Anticipation
```
Wouldn't worry about this since it's still (N) for now

Blitz425 replied:
Despite it being (N) it was intentional since in fight 1 it has Emergency Exit but in fight 2 I pictured it as “overcoming” its weakness. From when I tested it, it was really strong without the forced swap gimmick that I’m even considering giving it to fight 1 since the initial combo I wanted in Emergency Exit + Regenerator passive doesn’t heal it past boss bars. I see no issue keeping it like this regardless that way I won’t have to go back and change it once someone decides to implement Anticipation.

Blitz425 commented:
Updated Trainer Pools / Teams one more time, after thinking it over a bit I decided Lusamine's team is fine as is. Also updated Archie once more since Kingdra lost Drizzle as a passive this new patch replaced with Empoleon who gained Drizzle this patch. All I'm waiting for now is translations to be done, and new sprites for Lusamine/Guzma/Rose which are being made. Otherwise I feel this is complete now.

RimKnight commented on src/locales/zh_CN/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
   "battle_flare_grunt": "XY「战斗！闪焰队」",
+  "battle_aether_grunt": "日月「战斗！以太基金会」",
+  "battle_skull_grunt": "日月「战斗！骷髅队」",
+  "battle_macro_grunt": "剑盾「战斗！训练师」",
+  "battle_galactic_admin": "BDSP Team Galactic Admin Battle",
+  "battle_skull_admin": "SM Team Skull Admin Battle",
+  "battle_oleana": "SWSH Oleana Battle",
```
```suggestion
  "battle_macro_grunt": "剑盾「战斗！马洛科蒙集团」",
  "battle_galactic_admin": "晶灿钻石·明亮珍珠「战斗！银河队干部」",
  "battle_skull_admin": "日月「战斗！骷髅队干部」",
  "battle_oleana": "剑盾「战斗！奥利薇」",
```

RimKnight commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunts": "闪焰队手下们",
+  "aether_grunt": "以太基金会职员",
+  "aether_grunt_female": "以太基金会职员",
+  "aether_grunts": "Aether Foundation Employees",
```
```suggestion
  "aether_grunts": "以太基金会职员",
```

RimKnight commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "骷髅队手下",
+  "skull_grunt_female": "骷髅队手下",
+  "skull_grunts": "Team Skull Grunts",
```
```suggestion
  "skull_grunts": "骷髅队手下",
```

RimKnight commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -156,6 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "马洛科蒙训练师",
+  "macro_grunt_female": "马洛科蒙训练师",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "macro_grunts": "马洛科蒙训练师",
```

RimKnight commented on src/locales/zh_CN/trainers.ts:
```diff
@@ -32,6 +35,9 @@ export const titles: SimpleTranslationEntries = {
   "flare_admin_female": "闪焰队干部",
+  "aether_admin": "以太基金会干部",
+  "skull_admin": "骷髅队干部",
+  "macro_admin": "马洛科蒙",
```
```suggestion
  "macro_admin": "马洛科蒙干部",
```

RimKnight reviewed (approved):
Update zh-ch translation

NicusPulcis commented on src/locales/it/trainers.ts:
```diff
@@ -19,6 +19,9 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Team Galactic Boss",
   "plasma_boss": "Team Plasma Boss",
   "flare_boss": "Team Flare Boss",
+  "aether_boss": "Aether President",
+  "skull_boss": "Team Skull Boss",
+  "macro_boss": "Macro Cosmos President",
```
```suggestion
  "aether_boss": "Direttrice Æther",
  "skull_boss": "Capo Team Skull",
  "macro_boss": "Presidente Macro Cosmos",
```

NicusPulcis commented on src/locales/it/trainers.ts:
```diff
@@ -32,6 +35,9 @@ export const titles: SimpleTranslationEntries = {
   "plasma_admin": "Team Plasma Admin",
   "flare_admin": "Team Flare Admin",
   "flare_admin_female": "Team Flare Admin",
+  "aether_admin": "Aether Foundation Admin",
+  "skull_admin": "Team Skull Admin",
+  "macro_admin": "Macro Cosmos",
```
```suggestion
  "aether_admin": "Capo Filiale Æther",
  "skull_admin": "Ufficiale Team Skull",
  "macro_admin": "Vicepresidente Macro Cosmos",
```

NicusPulcis commented on src/locales/it/trainers.ts:
```diff
@@ -156,7 +162,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunt_female": "Recluta Team Flare",
   "flare_grunts": "Reclute Team Flare",
-
+  "aether_grunt": "Aether Foundation Employee",
+  "aether_grunt_female": "Aether Foundation Employee",
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "Team Skull Grunt",
+  "skull_grunt_female": "Team Skull Grunt",
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "Macro Cosmos Trainer",
+  "macro_grunt_female": "Macro Cosmos Trainer",
+  "macro_grunts": "Macro Cosmos Trainers",
```
```suggestion
  "aether_grunt": "Dipendente (Fondazione Æther)",
  "aether_grunt_female": "Dipendente (Fondazione Æther)",
  "aether_grunts": "Dipendenti (Fondazione Æther)",
  "skull_grunt": "Recluta Team Skull",
  "skull_grunt_female": "Recluta Team Skull",
  "skull_grunts": "Reclute Team Skull",
  "macro_grunt": "Impiegato Macro Cosmos",
  "macro_grunt_female": "Impiegata Macro Cosmos",
  "macro_grunts": "Impiegati Macro Cosmos",
```

NicusPulcis commented on src/locales/it/trainers.ts:
```diff
@@ -302,12 +316,17 @@ export const trainerNames: SimpleTranslationEntries = {
   "rood": "Rood",
   "xerosic": "Xerosic",
   "bryony": "Bryony",
+  "faba": "Faba",
+  "plumeria": "Plumeria",
```
```suggestion
  "faba": "Vicio",
  "plumeria": "Plumeria",
```

NicusPulcis commented on src/locales/it/trainers.ts:
```diff
@@ -302,12 +316,17 @@ export const trainerNames: SimpleTranslationEntries = {
   "cyrus": "Cyrus",
   "ghetsis": "Ghetsis",
   "lysandre": "Lysandre",
+  "lusamine": "Lusamine",
+  "guzma": "Guzma",
+  "rose": "Rose",
```
```suggestion
  "lusamine": "Samina",
  "guzma": "Guzman",
  "rose": "Rose",
```

NicusPulcis reviewed (approved):
italian localization done for trainers names, i'm leaving BGM as it is and for dialogues i'll just translate them all in one sitting eventually. make sure to solve conflicts before merging trainers names, as i've made a PR to localize them all some time ago and with new lines added there might be some issues

AsdarDevelops commented on src/locales/es/dialogue.ts:
```diff
@@ -676,6 +676,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      5: "Now that you've learned of the darkness at the heart of Aether Paradise, we'll need you to conveniently disappear!"
+    },
+    "victory": {
+      1: "Hmph! You seem to have a lot of skill.",
+      2: "What does this mean? What does this mean!",
+      3: "Hey! You're so strong that there's no way I can turn you away!",
+      4: "Hmm... It seems as though I may have lost.",
+      5: "Here's an impression for you: Aiyee!"
```
```suggestion
      1: "¡Vaya! Demuestras habilidad.",
      2: "¿Qué quiere decir esto? ¿Eh?",
      3: "¡Vaya! Con esa fuerza, ¡no creo que pueda detenerte!",
      4: "Vaya... parece que puede que haya perdido.",
      5: "¡Mira, te hago una mueca!: ¡Aiyee!"
```

AsdarDevelops commented on src/locales/es/dialogue.ts:
```diff
@@ -676,6 +676,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+  },
+  "faba": {
+    "encounter": {
+      1: "I, Branch Chief Faba, shall show you the harshness of the real world!",
+      2: "The man who is called Aether Paradise's last line of defense is to battle a mere child?",
+      3: "I, Faba, am the Aether Branch Chief. The only one in the world, I'm irreplaceable."
```
```suggestion
      1: "¡Yo, Subdirector Fabio, te mostraré la cruda realidad del mundo!",
      2: "¿Yo, la última línea de defensa de Æther, battallando a un mero crío?", // TODO: esGender
      3: "Yo, Fabio, soy el Subdirector de la Fundación Æther. Soy el único en el mundo, soy irremplazable."
```

AsdarDevelops commented on src/locales/es/dialogue.ts:
```diff
@@ -676,6 +676,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      3: "I, Faba, am the Aether Branch Chief. The only one in the world, I'm irreplaceable."
+    },
+    "victory": {
+      1: "Aiyee!",
+      2: "H-h-how can this be?! How could this child...",
+      3: "This is why... This is why I can't bring myself to like children."
```
```suggestion
      1: "¡Aaaah!",
      2: "¿C-c-cómo puede ser? Eres tan solo un niño...", // TODO: esGender
      3: "Este... este es el motivo por el que odio a los niños."
```

AsdarDevelops commented on src/locales/es/dialogue.ts:
```diff
@@ -808,6 +888,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
   },
+  "aether_boss_lusamine_1": {
+    "encounter": {
+      1: "You're going to startle my sweet beast! It looks like I'll need to silence you first."
+    },
+    "victory": {
+      1: "How.. how can you be so awful!"
+    },
+    "defeat": {
+      1: "Hmph..."
+    }
+  },
+  "aether_boss_lusamine_2": {
+    "encounter": {
+      1: `Why must you continue to pester me?! I am sick of you. Sick through and through!
+          $Enough with this useless talk.. with Nihilego's power I will show you how wrong you were to come here!`
+    },
+    "victory": {
+      1: "Aaauuuggghhhhhhhhh!!!"
+    },
+    "defeat": {
+      1: "All that I want is my precious beast! I don't care about any of the rest of you!"
```
```suggestion
      1: "¡Vas a asustar a mi pobre criatura! Parece que tendré que silenciarte cuanto antes."
    },
    "victory": {
      1: "¿Cómo?¿Cómo puedes ser tan terrible?"
    },
    "defeat": {
      1: "Buff."
    }
  },
  "aether_boss_lusamine_2": {
    "encounter": {
      1: `¡¿Por qué me sigues molestando?! Estoy cansada de ti. ¡Hasta las narices!
          $Se acabó esta charla innecesaria. Este es el poder de Nihilego, ¡te demostraré cuánto te equivocas al venir aquí!`
    },
    "victory": {
      1: "¡¡¡Aaauuuggghhhhhhhhh!!!"
    },
    "defeat": {
      1: "¡Todo lo que quiero es a esta preciosa criatura! ¡Los demás no me importáis!"
```

AsdarDevelops commented on src/locales/es/trainers.ts:
```diff
@@ -19,6 +19,9 @@ export const titles: SimpleTranslationEntries = {
   "galactic_boss": "Team Galactic Boss",
   "plasma_boss": "Team Plasma Boss",
   "flare_boss": "Team Flare Boss",
+  "aether_boss": "Aether President",
```
```suggestion
  "aether_boss": "Presidente Æther",
```

AsdarDevelops commented on src/locales/es/trainers.ts:
```diff
@@ -32,6 +35,9 @@ export const titles: SimpleTranslationEntries = {
   "plasma_admin": "Team Plasma Admin",
   "flare_admin": "Team Flare Admin",
   "flare_admin_female": "Team Flare Admin",
+  "aether_admin": "Aether Foundation Admin",
```
```suggestion
  "aether_admin": "Director de la Fundación Æther", // TODO: esGender
```

AsdarDevelops commented on src/locales/es/trainers.ts:
```diff
@@ -155,6 +161,15 @@ export const trainerClasses: SimpleTranslationEntries = {
   "flare_grunt": "Flare Grunt",
   "flare_grunt_female": "Flare Grunt",
   "flare_grunts": "Flare Grunts",
+  "aether_grunt": "Aether Foundation Employee",
+  "aether_grunt_female": "Aether Foundation Employee",
+  "aether_grunts": "Aether Foundation Employees",
```
```suggestion
  "aether_grunt": "Empleado de la Fundación Æther",
  "aether_grunt_female": "Empleada de la Fundación Æther",
  "aether_grunts": "Empleados de la Fundación Æther",
```

AsdarDevelops commented on src/locales/es/trainers.ts:
```diff
@@ -300,13 +315,18 @@ export const trainerNames: SimpleTranslationEntries = {
   "rood": "Rood",
   "xerosic": "Xerosic",
   "bryony": "Bryony",
+  "faba": "Faba",
```
```suggestion
  "faba": "Fabio",
```

AsdarDevelops commented on src/locales/es/trainers.ts:
```diff
@@ -300,13 +315,18 @@ export const trainerNames: SimpleTranslationEntries = {
   "ghetsis": "Ghetsis",
   "lysandre": "Lysandre",
-
+  "lusamine": "Lusamine",
```
```suggestion
  "lusamine": "Samina",
```

AsdarDevelops reviewed (changes requested):
Added all the Aether stuff in ES

Adri1 commented:
French dialogues added as the [PR to your branch](https://github.com/Blitz425/pokerogue/pull/1)
(because I had to add the female version of the dialogues, and wasn't possible tru the suggestions)

josericardo-fo commented on src/locales/pt_BR/bgm-name.ts:
```diff
@@ -80,11 +80,20 @@ export const bgmName: SimpleTranslationEntries = {
+  "battle_macro_grunt": "SWSH Batalha de Treinador",
+  "battle_galactic_admin": "BDSP Batalha com Admin da Equipe Galáctica",
+  "battle_skull_admin": "SM Batalha com Admin da Euipe Skull",
+  "battle_oleana": "SWSH Oleana Battle",
```
```suggestion
  "battle_oleana": "SWSH Batalha da Oleana",
```

josericardo-fo commented on src/locales/en/dialogue.ts:
```diff
@@ -678,6 +678,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+  "plumeria": {
+    "encounter": {
+      1: " ...Hmph. You don't look like anything special to me.",
+      2: "It takes these dumb Grunts way too long to deal with you kids..",
```
I think this is incorrect.
[Here](https://bulbapedia.bulbagarden.net/wiki/Plumeria/Quotes) it quotes: "It takes these dumb Grunts way too long to deal with you kids..."

josericardo-fo commented on src/locales/en/dialogue.ts:
```diff
@@ -813,6 +893,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      1: "You're going to startle my sweet beast! It looks like I'll need to silence you first."
+    },
+    "victory": {
+      1: "How.. how can you be so awful!"
```
Same thing here.
https://bulbapedia.bulbagarden.net/wiki/Lusamine/Quotes

josericardo-fo commented on src/locales/pt_BR/dialogue.ts:
```diff
@@ -659,6 +659,86 @@ export const PGMdialogue: DialogueTranslationEntries = {
+      4: "Wha-?! This kid's way too strong-no bones about it!",
+      5: "So, what? I'm lower than a Pokémon?! I already got self-esteem issues, man."
+    },
+  },
+  "plumeria": {
+    "encounter": {
+      1: " ...Hmph. You don't look like anything special to me.",
+      2: "It takes these dumb Grunts way too long to deal with you kids..",
+      3: "Mess with anyone in Team Skull, and I'll show you how serious I can get."
+    },
+    "victory": {
+      1: "Hmmph! You're pretty strong. I'll give you that.",
+      2: "Hmmph. Guess you are pretty tough. Now I understand why my Grunts waste so much time battling kids.",
+      3: "Hmmph! I guess I just have to hold that loss."
+    },
+  },
+  "macro_grunt": {
+    "encounter": {
+      1: "It looks like this is the end of the line for you!",
+      2: "You are a trainer aren't you? I'm afraid that doesn't give you the right to interfere in our work.",
+      3: "I'm from Macro Cosmos Insurance! Do you have a life insurance policy?"
+    },
+    "victory": {
+      1: "I have little choice but to respectfully retreat.",
+      2: "Having to give up my pocket money... Losing means I'm back in the red...",
+      3: "Nobody can beat Macro Cosmos when it comes to our dedication to our work!"
+    },
+  },
+  "oleana": {
+    "encounter": {
+      1: "I won't let anyone interfere with Mr. Rose's plan!",
+      2: "So, you got through all of the special staff that I had ordered to stop you. I would expect nothing less.",
+      3: "For the chairman! I won't lose!"
+    },
+    "victory": {
+      1: "*sigh* I wasn't able to win... Oleana...you really are a hopeless woman.",
+      2: "Arghhh! This is inexcusable... What was I thinking... Any trainer who's made it this far would be no pushover..",
+      3: "*sigh* I am one tired Oleana..."
+    },
+  },
```
```suggestion
  "aether_grunt": {
    "encounter": {
      1: "Eu vou lutar com tudo o que tenho para te eliminar!",
      2: "Eu não me importo se você é uma criança ou o que. Eu vou te mandar voando se você nos ameaçar!",
      3: "Me mandaram afastar Treinadores, não importa quem sejam!",
      4: "Eu vou te mostrar o poder do Paraíso Aether!",
      5: "Agora que você descobriu a escuridão no coração do Paraíso Aether, precisamos que você desapareça convenientemente!"
    },
    "victory": {
      1: "Aff! Você parece ter muita habilidade.",
      2: "O que isso significa? O que isso significa!",
      3: "Ei! Você é tão forte que não há como eu te afastar!",
      4: "Hmm... Parece que eu posso ter perdido.",
      5: "Aqui está uma resposta para você: Aiyee!"
    },
  },
  "faba": {
    "encounter": {
      1: "Eu, o Chefe de Ramo Faba, vou te mostrar a dureza do mundo real!",
      2: "O homem chamado de última linha de defesa do Paraíso Aether vai batalhar com uma mera criança?",
      3: "Eu, Faba, sou o Chefe de Ramo do Paraíso Aether. O único no mundo, eu sou insubstituível."
    },
    "victory": {
      1: "Aiyee!",
      2: "C-c-como assim?! Como uma criança...",
      3: "Isso é por que... Isso é por que eu não consigo gostar de crianças."
    },
  },
  "skull_grunt": {
    "encounter": {
      1: "Nós não somos maus, somos apenas durões!",
      2: "Quer um pouco de nós? É assim que dizemos olá! Prazer em conhecê-los, otários!",
      3: "Somos só um bando de caras e minas com um grande interesse nos Pokémon dos outros!",
      4: "Por que você está tentando agir durão quando já somos durões como ossos aqui fora, mano?",
      5: "Equipe Skull na área! Não podemos pagar o aluguel! Nos divertimos muito, mas nossa juventude foi desperdiçada!"
    },
    "victory": {
      1: "Ahn? Já acabou?",
      2: "Hora de dar o fora, mano! Vamos ter que dizer tchau, mano!",
      3: "Nem precisamos dos seus Pokémon fracotes mesmo!",
      4: "Que?! Esse muleque é muito forte! Não tem como negar!",
      5: "Então, o que? Eu sou pior que um Pokémon?! Eu já tenho problemas de autoestima, cara."
    },
  },
  "plumeria": {
    "encounter": {
      1: " ...Hmph. Você não parece nada especial para mim.",
      2: "Esses Capangas gastam muito tempo tentando lidar com crianças como você...",
      3: "Mexe com qualquer um da Equipe Skull, e eu vou te mostrar o quão séria eu posso ficar."
    },
    "victory": {
      1: "Aff! Você é bem forte. Tenho que admitir.",
      2: "Aff! Você é bem forte. Agora eu entendi por que meus Capangas gastam tanto tempo batalhando com crianças.",
      3: "Aff! Acho que eu só tenho que aceitar essa derrota."
    },
  },
  "macro_grunt": {
    "encounter": {
      1: "Parece que aqui é o fim da linha para você!",
      2: "Você é um treinador, não é? Temo que isso não lhe dê o direito de interferir em nosso trabalho.",
      3: "Sou da Macro Cosmos Seguros! Já tem um seguro de vida?"
    },
    "victory": {
      1: "Eu não tenho muita escolha a não ser recuar respeitosamente.",
      2: "Ter que desistir do meu dinheiro... Perder significa que estou de volta no vermelho...",
      3: "Ninguém pode vencer a Macro Cosmos quando se trata de nossa dedicação ao trabalho!"
    },
  },
  "oleana": {
    "encounter": {
      1: "Eu não vou deixar ninguém interferir no plano do Sr. Rose!",
      2: "Então, você passou por todo o pessoal especial que eu ordenei para te parar. Eu não esperaria nada menos.",
      3: "Pelo presidente! Eu não vou perder!"
    },
    "victory": {
      1: "*suspiro* Eu não consegui vencer... Oleana... você realmente é uma mulher sem esperança.",
      2: "Aff! Isso é inaceitável... O que eu estava pensando... Qualquer treinador que chega tão longe não seria um fracote...",
      3: "*suspiro* Eu sou uma Oleana cansada..."
    },
  },
```

josericardo-fo commented on src/locales/pt_BR/dialogue.ts:
```diff
@@ -791,6 +871,82 @@ export const PGMdialogue: DialogueTranslationEntries = {
+    "encounter": {
+      1: `Doesn't matter who I'm up against, I'm gonna beat them down!
+          $That's what big bad Guzma is all about!`
+    },
+    "victory": {
+      1: "Guzma!!! What is wrong with you?!"
+    },
+    "defeat": {
+      1: "Y'all are stupid!"
+    }
+  },
+  "macro_boss_rose_1": {
+    "encounter": {
+      1: `I must provide limitless energy to ensure everlasting prosperity for everyone...
+          $It is my purpose, my duty, my destiny!`
+    },
+    "victory": {
+      1: `You still don't understand, trainer...
+          $We... No, I am going to change the course of history!`
+    },
+    "defeat": {
+      1: "You still don't understand a thing!"
+    }
+  },
+  "macro_boss_rose_2": {
+    "encounter": {
+      1: `I'm committed to solving the energy problem in the Galar region—and, of course, around the world.
+          $My experience and accomplishments that made Macro Cosmos a success are proof my methods work.
+          $I don't intend to change my mind, even if I lose.`
+
+    },
+    "victory": {
+      1: `I'd forgotten how great Pokémon battles are! It's been so long since I battled...
+          $That sure was satisfying, I accept defeat for this battle.`
+    },
+    "defeat": {
+      1: `I suppose it must seem that I am doing something terrible. I don't expect you to understand.
+          $But I must provide the Galar region with limitless energy to ensure everlasting prosperity.`
+    }
+  },
```
```suggestion
  "aether_boss_lusamine_1": {
    "encounter": {
      1: "Você vai assustar minha fera querida! Parece que eu vou precisar te silenciar primeiro."
    },
    "victory": {
      1: "Como... como você pode ser tão terrível!"
    },
    "defeat": {
      1: "Aff..."
    }
  },
  "aether_boss_lusamine_2": {
    "encounter": {
      1: `Por que você continua a me incomodar?! Estou cansada de você. Cansada até a alma!
          $Chega de papo furado... Com o poder de Nihilego, vou mostrar o quão errado você estava em vir aqui!`
    },
    "victory": {
      1: "Aaauuuggghhhhhhhhh!!!"
    },
    "defeat": {
      1: "Tudo o que eu quero é minha fera preciosa! Eu não me importo com nenhum de vocês!"
    }
  },
  "skull_boss_guzma_1": {
    "encounter": {
      1: `O odiado chefe que te bate e te bate e nunca para...
          $É. O grande e mau Guzma está aqui!`
    },
    "victory": {
      1: "Tss. Eu vou te bater um dia desses!"
    },
    "defeat": {
      1: "E você veio até aqui só pra isso, né?"
    }
  },
  "skull_boss_guzma_2": {
    "encounter": {
      1: `Não importa contra quem eu esteja, eu vou derrotá-los!
          $Isso é o que o grande e mau Guzma é!`
    },
    "victory": {
      1: "Guzma!!! O que há de errado com você?!"
    },
    "defeat": {
      1: "Vocês são tudo uns idiotas!"
    }
  },
  "macro_boss_rose_1": {
    "encounter": {
      1: `Devo fornecer energia ilimitada para garantir a prosperidade eterna para todos...
          $É meu propósito, meu dever, meu destino!`
    },
    "victory": {
      1: `Você ainda não entendeu, treinador...
          $Nós... Não, eu vou mudar o rumo da história!`
    },
    "defeat": {
      1: "Você ainda não entende nada!"
    }
  },
  "macro_boss_rose_2": {
    "encounter": {
      1: `Estou comprometido em resolver o problema de energia na região de Galar e, claro, em todo o mundo.
          $Minha experiência e conquistas que fizeram da Macro Cosmos um sucesso são a prova de que meus métodos funcionam.
          $Não pretendo mudar de ideia, mesmo que eu perca.`
    },
    "victory": {
      1: `Eu esqueci o quão incríveis são as batalhas Pokémon! Faz tanto tempo desde que eu batalhei...
          $Isso foi realmente satisfatório, eu aceito a derrota para esta batalha.`
    },
    "defeat": {
      1: `Eu suponho que deve parecer que estou fazendo algo terrível. Eu não espero que você entenda.
          $Mas eu devo fornecer à região de Galar energia ilimitada para garantir prosperidade eterna.`
    }
  },
```

josericardo-fo commented:
Could you please add these translations to the female dialogues for latin languages?

CodeTappert commented on src/locales/de/trainer-classes.json:
```diff
@@ -117,5 +117,14 @@
+  "aether_grunts": "Aether Foundation Employees",
+  "skull_grunt": "Rüpel von Team Skull",
+  "skull_grunt_female": "Rüpel von Team Skull",
+  "skull_grunts": "Team Skull Grunts",
```
```suggestion
  "skull_grunts": "Rüpel von Team Skull",
```

CodeTappert commented on src/locales/de/trainer-classes.json:
```diff
@@ -117,5 +117,14 @@
+  "skull_grunts": "Team Skull Grunts",
+  "macro_grunt": "Angestellter von Macro Cosmos",
+  "macro_grunt_female": "Angestellte von Macro Cosmos",
+  "macro_grunts": "Macro Cosmos Trainers"
```
```suggestion
  "macro_grunts": "Angestellte von Macro Cosmos"
```

CodeTappert commented on src/locales/de/trainer-classes.json:
```diff
@@ -117,5 +117,14 @@
+  "flare_grunts": "Rüpel von Team Flare",
+  "aether_grunt": "Æther Foundation Personal",
+  "aether_grunt_female": "Æther Foundation Personal",
+  "aether_grunts": "Aether Foundation Employees",
```
```suggestion
  "aether_grunts": "Æther Foundation Personal",
```

CodeTappert reviewed (approved):
Review as Dev and as Translation Team
(Reviewd the code. Did NOT do a playtest since i am at work!)

damocleas reviewed (dismissed):
Approving on Balance side (says from one balance member to another despite looking over most of it as it was being made)

damocleas reviewed (approved):
dude

Madmadness65 reviewed (approved):
Kinda funky how GitHub shows that the entirety of the trainer config file was replaced, but it _appears_ to be fine from my testing. In any case, this lgtm, the new teams seem to be appearing as intended.
